<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>product_information_window</class>
 <widget class="QDialog" name="product_information_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#product_information_window {
	background-image : url(:/Images/ProductInformationWindow.png);
}</string>
  </property>
  <widget class="QLabel" name="Lb_exDate">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>330</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: Calibri;
font-size: 17px;
color: #38446d;
font-weight: bold;
text-transform: none;</string>
   </property>
   <property name="text">
    <string>2025\12\11</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>300</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: Calibri;
font-size: 14px;
color: #38446d;
font-weight: bold;
text-transform: none;</string>
   </property>
   <property name="text">
    <string>Expration Date</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: Calibri;
font-size: 14px;
color: #38446d;
font-weight: bold;
text-transform: none;</string>
   </property>
   <property name="text">
    <string>SKU</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>380</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: Calibri;
font-size: 14px;
color: #38446d;
font-weight: bold;
text-transform: none;</string>
   </property>
   <property name="text">
    <string>Price</string>
   </property>
  </widget>
  <widget class="QLabel" name="LB_inventory">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>120</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: Calibri;
font-size: 17px;
color: #38446d;
font-weight: bold;
text-transform: none;</string>
   </property>
   <property name="text">
    <string>983 Kg</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: Calibri;
font-size: 14px;
color: #38446d;
font-weight: bold;
text-transform: none;</string>
   </property>
   <property name="text">
    <string>Brand</string>
   </property>
  </widget>
  <widget class="QLabel" name="LB_name">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: Calibri;
font-size: 17px;
color: #38446d;
font-weight: bold;
text-transform: none;</string>
   </property>
   <property name="text">
    <string>Product Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="LB_brand">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: Calibri;
font-size: 17px;
color: #38446d;
font-weight: bold;
text-transform: none;</string>
   </property>
   <property name="text">
    <string>Product Brand</string>
   </property>
  </widget>
  <widget class="QLabel" name="LB_selected">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>200</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: Calibri;
font-size: 17px;
color: #38446d;
font-weight: bold;
text-transform: none;</string>
   </property>
   <property name="text">
    <string>432</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>240</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: Calibri;
font-size: 14px;
color: #38446d;
font-weight: bold;
text-transform: none;</string>
   </property>
   <property name="text">
    <string>Added Date</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: Calibri;
font-size: 14px;
color: #38446d;
font-weight: bold;
text-transform: none;</string>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="LB_price">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>400</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: Calibri;
font-size: 17px;
color: #38446d;
font-weight: bold;
text-transform: none;</string>
   </property>
   <property name="text">
    <string>$ 123.32</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>360</y>
     <width>258</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_22">
      <property name="styleSheet">
       <string notr="true">font-family: Calibri;
font-size: 14px;
color: #38446d;
font-weight: bold;
text-transform: none;</string>
      </property>
      <property name="text">
       <string>Description</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="descriptionTextBrowser">
      <property name="styleSheet">
       <string notr="true">border : none;
font-family: Calibri;
font-size: 12px;
color: #38446d;
background-color : #D7DBE7;</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Calibri'; font-size:12px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Segoe UI'; font-size:9pt;&quot;&gt;This is a ....&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Segoe UI'; font-size:9pt;&quot;&gt;Sample Description&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: Calibri;
font-size: 14px;
color: #38446d;
font-weight: bold;
text-transform: none;</string>
   </property>
   <property name="text">
    <string>Category</string>
   </property>
  </widget>
  <widget class="QLabel" name="LB_addDate">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>270</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: Calibri;
font-size: 17px;
color: #38446d;
font-weight: bold;
text-transform: none;</string>
   </property>
   <property name="text">
    <string>2023\12\12</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: Calibri;
font-size: 18px;
color: #47386d;
font-weight: bold;
text-transform: none;</string>
   </property>
   <property name="text">
    <string>Product Information</string>
   </property>
  </widget>
  <widget class="QLabel" name="LB_category">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>270</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: Calibri;
font-size: 17px;
color: #38446d;
font-weight: bold;
text-transform: none;</string>
   </property>
   <property name="text">
    <string>Product category</string>
   </property>
  </widget>
  <widget class="QLabel" name="LB_sku">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>340</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: Calibri;
font-size: 17px;
color: #38446d;
font-weight: bold;
text-transform: none;</string>
   </property>
   <property name="text">
    <string>Product SKU</string>
   </property>
  </widget>
  <widget class="QLabel" name="LB_inventoryLabel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>90</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: Calibri;
font-size: 14px;
color: #38446d;
font-weight: bold;
text-transform: none;</string>
   </property>
   <property name="text">
    <string>Inventory</string>
   </property>
  </widget>
  <widget class="QLabel" name="LB_selectedLabel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>170</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: Calibri;
font-size: 14px;
color: #38446d;
font-weight: bold;
text-transform: none;</string>
   </property>
   <property name="text">
    <string>Selected</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
